packages/contracts-bedrock/src/L2/CrossL2Inbox.sol

```diff
@@ -174,19 +131,9 @@ contract CrossL2Inbox is ISemver, TransientReentrancyAware {
         // We need to know if this is being called on a depositTx
         if (IL1BlockInterop(Predeploys.L1_BLOCK_ATTRIBUTES).isDeposit()) revert NoExecutingDeposits();
 
-        // Check the Identifier.
-        _checkIdentifier(_id);
-
         emit ExecutingMessage(_msgHash, _id);
     }
 
-    /// @notice Validates that for a given cross chain message identifier
-    ///         it's timestamp is not in the future.
-    /// @param _id Identifier of the message.
-    function _checkIdentifier(Identifier calldata _id) internal view {
-        if (_id.timestamp > block.timestamp || _id.timestamp <= interopStart()) revert InvalidTimestamp();
-    }
-
     /// @notice Stores the Identifier in transient storage.
     /// @param _id Identifier to store.
     function _storeIdentifier(Identifier calldata _id) internal {
```

Comment:
We chatted about this out of band, but we can remove the whole `_storeIdentifier` function and the getters for the context variables given the only entrypoint now is the `validateMessage` function